creation/web_base/cvmfs_helper_funcs.sh

```diff
@@ -160,7 +160,7 @@ perform_system_check() {
 	# call function to detect local CVMFS only if the GWMS_IS_CVMFS_MNT variable is not set; if the variable is not empty, do nothing
 	[[ -z "${GWMS_IS_CVMFS_MNT}" ]] && detect_local_cvmfs || :
 
-	sysctl user.max_user_namespaces &>/dev/null
+    cat /proc/sys/user/max_user_namespaces &>/dev/null
 	GWMS_IS_UNPRIV_USERNS_SUPPORTED=$?
```

Comment:
Is the presence of the file or the value telling you if unpriv user namespace is enabled?
I think it may be the value. `$?` is the exit code

Reply:
Thanks for pointing this out. You are correct in thinking that it is the value that is useful in determining if unpriv. user namespaces is supported; for checking whether unpriv. user namespaces are enabled, the `unshare -U true` command is used.

Dave had also mentioned about checking for value being the correct approach to determine the status of unpriv. user namespaces, but it had completely skipped my mind. I have updated the code to reflect your (and Dave's) suggestion.